### PR TITLE
预览文本支持GB2312

### PR DESCRIPTION
**Pull request by a57495681800**
在预览时新增了一个Markdown_GB2312选项
[issue](https://github.com/alist-org/alist/issues/4131)

### CODE REVIEW

File: src/lang/en/drivers.json [hunk 1]
```diff
@@ -363,6 +363,7 @@
     "password": "Password",
     "phone": "Phone",
     "root_folder_id": "Root folder id",
+    "root_folder_id-tips": "be careful when using the -11 value, some operations may cause system errors",
```
Comment (xhofe) [inline src/lang/en/drivers.json]: what's this??

File: src/hooks/useUtil.ts [hunk 2]
```diff
@@ -25,10 +25,10 @@ export const useUtil = () => {
   }
 }
 
-export const useFetchText = () => {
+export const useFetchText = (flag = false) => {
```
Comment (xhofe) [inline src/hooks/useUtil.ts]: you can use a more meaningful variable name

File: src/components/Markdown-GB2312.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// @ts-ignore
```
Comment (xhofe) [inline src/components/Markdown-GB2312.tsx]: This component seems to be unnecessary

File: src/utils/api.ts [hunk 4]
```diff
@@ -113,9 +113,28 @@ export const offlineDownload = (
   return r.post(`/fs/add_${type}`, { path, urls })
 }
 
+export const helper = async (url: string) =>{
```
Comment (xhofe) [inline src/utils/api.ts]: you can move this function to `/utils/str.ts`

Comment (xhofe) [inline src/utils/api.ts]: you can move this function to `/utils/str.ts`

**Review by xhofe — CHANGES_REQUESTED** [review]

File: src/utils/api.ts (hunk 4 above)
Comment (a57495681800) [reply src/utils/api.ts]: fixed

**Review by xhofe — APPROVED** [review]

**Comment by xhofe** [comment]
Thanks for your contribution!

**Comment by a57495681800** [comment]
You're welcome

**Pull request merged**